.github/workflows/trivy.yml

```
@@ -0,0 +1,10 @@
+name: build
+on:
+  pull_request:
+jobs:
+  build:
+    name: Security Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Trivy Security check on Repository
+        uses: it-at-m/lhm_actions/action-templates/actions/action-trivy
```

actionlint (1.7.12)

[error] specifying action "it-at-m/lhm_actions/action-templates/actions/action-trivy" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)